jobs/config.json

```diff
@@ -10564,27 +10567,6 @@
       "sig-aws"
     ]
   },
-  "pull-kubernetes-e2e-kops-aws-prow": {
```

Comment:
you don't want to keep the canary job?

Reply:
(make canary -> canary pass -> flip canary to prod, I think we don't need the canary anymore)